### PR TITLE
doc: instructions for joining and leaving the network as a validator

### PR DESCRIPTION
**Pull request by Windforcer**

### CODE REVIEW

**Comment by Windforcer** [comment]
I will be around a few hours before the ceremony to implement any feedback

**Review by cgorenflo — APPROVED** [review]

**Review by sergeynog — CHANGES_REQUESTED** [review]

File: Exercises/excercise_3_join_leave_validator.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    --commission-rate="0.10" \
+    --commission-max-rate="0.20" \
+    --commission-max-change-rate="0.01" \
+    --min-self-delegation="1" \
+    --pubkey "$(axelard tendermint show-validator)" \
+    --from validator \
+    -b block
+  ```
+
+  Here `amount` refers to the amount of coins to stake, with a minimum of 500,000 axltest. You can change the amount, but leave some coins on the account to fund commands.
```
[the remark this answers is not in the document]
Comment (cgorenflo) [reply Exercises/excercise_3_join_leave_validator.md:95]: correct

**Comment by Windforcer** [comment]
Thanks for the comments, I've implemented the suggestions :)

File: Exercises/excercise_3_join_leave_validator.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sergeynog) [reply Exercises/excercise_3_join_leave_validator.md:92]: following up on our offline discussion with Chris, we need `50000000` to become active for keygen/sign (10B is total supply). i would put that here. we'll update the faucet to distribute more. @Windforcer @cgorenflo

[the remark this answers is not in the document]
Comment (Windforcer) [reply Exercises/excercise_3_join_leave_validator.md]: Staking amount increased by 10x in instructions

**Review by sergeynog — APPROVED** [review]

**Pull request merged**